app/lib/scorecard/models.dart

```diff
@@ -245,6 +246,32 @@ class PanaReport {
     return PanaReport.fromJson(map);
   }
 
+  static PanaReport fromSummary(
+    Summary? summary, {
+    required PackageStatus packageStatus,
+  }) {
+    final reportStatus =
+        summary == null ? ReportStatus.aborted : ReportStatus.success;
+    return PanaReport(
+      // TODO: use summary.createdAt once it lands
+      timestamp: clock.now().toUtc(),
```

Comment:
This will be very wrong when it's not stored in scorecard. So let's land the pana fix soon.

Reply:
Yeah, let's land that first...